setup/setup.sh

```diff
@@ -18,38 +18,51 @@
 
 
 function err() {
-  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+  echo "ERROR: $*" >&2
   return 1
 }
 
-
 function export_variables(){
     # Set variables.
     export RUNTIME_VERSION=1.13
     export PYTHON_VERSION=3.5
     export REGION=us-central1
 
-    # Replace "your-gcp-project-id" with your GCP PROJECT ID
-    export PROJECT_ID="your-gcp-project-id"
```

Comment:
can we simplify IF and ELSE in a function?

Reply:
What exactly do you have in mind?